command/init.go

```diff
@@ -541,15 +541,38 @@ func (c *InitCommand) getProviders(earlyConfig *earlyconfig.Config, state *state
 			}
 		},
 		FetchPackageSuccess: func(provider addrs.Provider, version getproviders.Version, localDir string, authResult *getproviders.PackageAuthenticationResult) {
-			var warning string
+			var (
```

Comment:
This is a nitpick, but I find single-line variable declarations easier to read when inside a function:

`var keyID, warning string`